lib/index.d.ts

```diff
@@ -42,6 +42,7 @@ export type FormState<Inf = Record<string, any>> = {
   updateValidateField: (field: keyof Inf, value: any) => Promise<any>;
   updateTouched: (field: keyof Inf, value: any) => void;
   validateField: (field: keyof Inf) => Promise<any>;
+  validateForm: () => Promise<any>;
```

Comment:
Not one for this PR, but it would probably be helpful to have stronger typing on this library. I think it would nice if we enforced the same kind of type standards as tw-api-js. I know that often we don't know the types that would be passed in to some of these functions, but we could probably use generics to work it out